src/api/app/models/workflow/step.rb

```diff
@@ -142,7 +149,7 @@ def report_to_scm(workflow_filters)
       # TODO: Fix n+1 queries
       workflow_architectures(repository, workflow_filters).each do |architecture|
         # We cannot report multibuild flavors here... so they will be missing from the initial report
-        SCMStatusReporter.new({ project: target_project_name, package: target_package_name, repository: repository.name, arch: architecture.name },
+        SCMStatusReporter.new({ project: target_project_name, package: target_package_name(short_commit_sha: true), repository: repository.name, arch: architecture.name },
```

Comment:
Setting `short_commit_sha` to `true` also needs to happen in `ReportToScmJob`. The package is in `@event.payload`: https://github.com/openSUSE/open-build-service/blob/0cb661478d70ecda31c6f9b6c052144c53e7b9bf/src/api/app/jobs/report_to_scm_job.rb#L11

Reply:
Exactly, I already talked about this with Dani. We are reporting with a short commit sha the first time we call `SCMStatusReporter` to set the "pending" state. The following reports, after a build is finished, will contain a long commit sha.

Reply:
Fixed, now